storage/innobase/xtrabackup/doc/source/innobackupex/replication_ibk.rst

```diff
@@ -1,23 +0,0 @@
-============================================
- Taking Backups in Replication Environments
```

Comment:
this section is valid for xtrabackup and the " How to setup a slave for replication in 6 simple steps with Percona XtraBackup" one doesn't seem to fully replace it

Reply:
Moved to xtrabackup_bin